docs/src/introduction.md

```diff
@@ -0,0 +1,247 @@
+# Getting Started with QuantumSymbolics.jl
+
+```@meta
+DocTestSetup = quote
+    using QuantumSymbolics
+end
+```
+
+QuantumSymbolics is designed for manipulation and numerical translation of symbolic quantum objects. This tutorial introduces basic features of the package.
+
+## Installation
+
+QuantumSymbolics.jl can be installed through the Julia package system in the standard way:
+
+```@example 1
+using Pkg
+Pkg.add("QuantumSymbolics")
+```
+
+## Literal Symbolic Quantum Objects
+
+Basic objects of type `SBra`, `SKet`, and `SOperator` represent symbolic quantum objects with `name` and `basis` properties. Each type can be generated with a straightforward macro:
+
+```jldoctest
+julia> using QuantumSymbolics
+
+julia> @bra b # object of type SBra
+⟨b|
+
+julia> @ket k # object of type SKet
+|k⟩
+
+julia> @op A # object of type SOperator
+A
+```
+
+By default, each of the above macros defines a symbolic quantum object in the spin-1/2 basis. One can simply choose a different basis, such as the Fock basis or a tensor product of several bases, by passing an object of type `Basis` to the second argument in the macro call:
+
+```jldoctest
+julia> @op B FockBasis(5)
+B
+
+julia> basis(B)
+Fock(cutoff=5)
+
+julia> @op C SpinBasis(1//2)⊗SpinBasis(5//2)
+C
+
+julia> basis(C)
+[Spin(1/2) ⊗ Spin(5/2)]
+```
+Here, we extracted the basis of the defined symbolic operators using the `basis` function.
+
+
+Symbolic quantum objects with additional properties can be defined, such as a Hermitian operator, or the zero ket (i.e., a symbolic ket equivalent to the zero vector $\bm{0}$).
```

Comment:
Did you have a chance to render the docs locally to check if this latex expression is rendered correctly?

You can run the local compilation with 
```
julia --project=docs -e '
            using Pkg
            Pkg.develop(PackageSpec(path=pwd()))
            Pkg.instantiate()';
julia --project=docs -i docs/make.jl
```

Reply:
Yes I did, it renders correctly.